onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/DmlDFT.h

```diff
@@ -0,0 +1,695 @@
+#pragma once
```

Comment:
🤔 It's kinda weird putting this all inside a header file given every other DML operator kernel is in a cpp, especially considering this includes another quite large header file full of shader data (which would lead to multiply defined symbols if not careful), and it also slows down every recompile of OperationRegistration.cpp more so. Can we move it to a cpp?

Reply:
This is a short lived implementation which should be moved into DML quickly. So for now I will leave it, as the expectation is that it will be removed soon.

Reply:
Ok.